### DISCUSSION TITLE
How does scope work?

### DISCUSSION THREAD

**Discussion by thasmo** [vuejs/petite-vue#10] [Q&A]
Is this intended? https://codepen.io/thasmo/pen/abWvPbo

I was under the impression that only data passed to `createApp` would be used.

**Comment by xunmi1** [comment]
https://github.com/vuejs/petite-vue/blob/696c6d6784df90cbd6a32c1e4e47a7ed172f12c6/src/app.ts#L42
https://github.com/vuejs/petite-vue/blob/696c6d6784df90cbd6a32c1e4e47a7ed172f12c6/src/app.ts#L51-L53

**Reply by xunmi1** [reply]
Probably so.

**Comment by yyx990803** [comment]
Yes, currently expressions are evaluated with access to global scope (i.e. `window`). This may change in the future so it's still best to explicitly pass properties to `createApp`. Also because in `<script type="module">` declared variables are not global (unlike in normal `<script>`.
